### PR TITLE
Notes on how to set up milestones in Jira

### PR DESCRIPTION
**Pull request by drphilmarshall**
New subsection on how to define milestones in Jira. I think these notes are accurate for how the system is currently set up, but I left comments marked "PJM" in a couple of places where we might want to provide firmer guidance (and hence push the system towards a different set-up). 

Cheers

Phil

### CODE REVIEW

File: body.tex [hunk 1]
```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
 agile way, while also ensuring that scope, cost and schedule are
 carefully controlled.
 
+
+\subsection{Defining the Schedule with Milestones}
+
+Rubin Milestones are defined as Jira issues of type ``Milestone''. As indicated above, the Director (or their designate) defines the L1 milestones, the Associate Directors (ADs) define their departments' L2 milestones, and the Team Leaders and ADs define the L3 milestones for their teams.
```
Comment (astropixie) [inline body.tex:237]: We do the initial Milestone defining in smartsheet (this is the tool we will use to do FY milestone planning.  Once all the milestones are input and ADs and team leads have coordinated and the Director approves, then Cathy creates the Jira Milestone Issues using the info in the smartsheet.

File: body.tex [hunk 2]
```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
...
+
+The following fields must be filled out when defining a new milestone:
```
Comment (astropixie) [inline body.tex:239]: Once Cathy has created the milestone issues in Jira, then the ADs and Team leads go and the full description.

File: body.tex [hunk 3]
```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
...
+  The ID number may get edited by the Director's Office to ensure that it is unique: the Milestone IDs are only ever used in linking together the milestones for visualization in Smartsheet. 
+  Good practice here is to look at the milestone list in Smartsheet and choose an ID number that leads to a milestone ID that has not already been taken, and which roughly fits your milestone into the time-ordered list.  
+\item
+  The \texttt{Summary} field should contain a sentence outlining either what needs to be done, or what will be delivered for the Milestone to be reached. 
+  Examples include \textit{``Deliver Data Preview 0.1 (DP0.1)''} (an L1 milestone) and \textit{``DP0.1 Data Release: science-ready catalogs released from the IDF''} (an L2 milestone that belongs to it).
+  % PJM: Should we homogenize the style here, and make Milestone summaries EITHER directives OR deliverables? Discussed with Amanda: leave this up to the teams' preferences.
+\item  
+  The \texttt{POP Activity Description} text should contain one or maybe two sentences summarizing the activity needed to complete the milestone.
```
Comment (astropixie) [inline body.tex:254]: why specify "POP" here if we are doing the same thing for all milestones (even non-Pop milestones)?   I think that's confusing.

File: body.tex [hunk 4]
```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
...
+  Example: \textit{``Upgrade the WBS activity, labor and non-labor plans from V4 to V5 in order to capture a US DF at SLAC, a UK DF, and any other modifications needed, and estimate the corresponding budget.''} 
+  Note that only L1 (and sometimes L2) milestones are actually listed in the NOIRLab Program Operations Plan (POP), but Rubin adopts the same structure for all its milestones.
+\item
+  The \texttt{POP Deliverable Description} is a very terse list of the deliverables needed to reach the Milestone. 
+  Example: \textit{``V5 WBS workbook and Preliminary Cost Calculator.''}
+\item 
+  The \texttt{Due Date} is the latest date in the future by which the milestone needs to be reached. 
+  This date should be before or the same as the milestone's parent milestone's ``Rubin Forecast Due Date'' as shown in the Smartsheet. 
+\item
+  The \texttt{Start Date} is the date when the work for the milestone should begin.
```
Comment (astropixie) [inline body.tex:264]: At the moment we are not requiring a start date - it is assumed to be at the start of a quarter or planning period.  An issue is that Leanne's group has already start work on FY22 milestones, which haven't been formally introduced to smartsheet or had Jira issues created yet.

**Review by astropixie — CHANGES_REQUESTED** [review]
Hi Phil.   Thanks for taking a first pass at the milestone descriptions.   I think your overall picture is correct, but Cathy and I had envisioned a different workflow for how this all gets done, under the assumption that the bulk of milestone definition gets done in the period of FY planning.  

I'm happy to accept your changes and then edit the text, but I'll give you the chance to address those issue first if you want.

**Comment by drphilmarshall** [comment]
Thanks Amanda! Before I jump in and address the specific comments, let me
check. I didn't think we were setting all of the L3 milestones ahead of
time with the Director's approval, and since the majority of milestones are
L3, I figured the technote should provide instructions for team leaders.
Those instructions are what I was missing, as I was playing the part of
surrogate Directorate team leader while making the V5 WBS L3 milestones. I
had to make an L2 milestone then as well, which I think that was somewhat
unusual, but still, I think there will be cases in future when an AD needs
to make an L2 milestone during the year. Do we need the technote to explain
the two different modes of milestone creation, during annual planning and
midstream?

File: body.tex (hunk 3 above)
Comment (drphilmarshall) [reply body.tex:254]:
```suggestion
  The \texttt{Activity Description} text should contain one or maybe two sentences summarizing the activity needed to complete the milestone. 
```

Comment (drphilmarshall) [reply body.tex:254]: We can drop the "POP" from the text here, but in Jira the name of the field is literally "POP Activity Description" for all issues of type "milestone", even if they are Level 2 or Level 3. We should change that to make the field simply "Activity Description" I think. Paging @cathypetry !

File: body.tex (hunk 4 above)
Comment (drphilmarshall) [reply body.tex:264]: I'm sure we can backfill when needed. The Jira milestone issues do have a Start Date field, which I guess we are treating as optional: if the creator doesn't enter anything, it sounds like we have a reasonable algorithm for defining a start date in Smartsheet automatically. I'll try and make this clear in the text.

File: body.tex (hunk 1 above)
Comment (drphilmarshall) [reply body.tex:237]: I added a note on this - the instructions for milestone issue creation are still needed I think, if only for a future Program Coordinator!

**Pull request merged**